ztp/siteconfig-generator/siteConfig/testdata/extra-manifest/01-predefined-mc-master.yaml

```diff
@@ -0,0 +1,73 @@
+# Automatically generated by extra-manifests-builder
```

Comment:
For these test files, I'd recommend going simpler and smaller.  And they're not generated by extra-manifest-builder, so we can remove this header too.